nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment-rest-api/src/test/java/org/nuxeo/ecm/restapi/server/jaxrs/comment/AnnotationAdapterTest.java

```diff
@@ -97,7 +97,7 @@ public void testCreateAnnotation() throws IOException {
                 jsonAnnotation)) {
             assertEquals(Response.Status.CREATED.getStatusCode(), response.getStatus());
             JsonNode node = mapper.readTree(response.getEntityInputStream());
-            assertEquals(file.getId(), node.get("documentId").textValue());
+            assertEquals(file.getId(), node.get("parentId").textValue());
             assertEquals(xpath, node.get("xpath").textValue());
```

Comment:
why not use COMMENT_PARENT_ID_FIELD in this file ?

Reply:
This test was there before introducing the constants. Will do the refactoring.